src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+    public String getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = colorSupplier.getRandomColor();
+        return randomColor;
```

Comment:
The method name 'getRandomBall' suggests that it should return a 'Ball' object, but it currently returns a String representing a color. Consider returning a new Ball object with a random color and a random number.